src/pages/DetailsPage.js

```diff
@@ -91,23 +91,34 @@ const DetailsPage = ({
                                 imageStyles={[styles.avatarLarge]}
                                 source={details.avatar}
                             />
-                            <Text style={[styles.displayName, styles.mt1, styles.mb6]} numberOfLines={1}>
-                                {details.displayName && isSMSLogin
-                                    ? toLocalPhone(details.displayName)
-                                    : (details.displayName || null)}
-                            </Text>
+                            <TappableCopy
+                                style={[styles.mt1, styles.mb6]}
+                                type={details.displayName && isSMSLogin ? 'phone' : undefined}
+                                value={getPhoneNumber(details)}
+                            >
+                                <Text style={[styles.displayName]} numberOfLines={1}>
+                                    {details.displayName && isSMSLogin
+                                        ? toLocalPhone(details.displayName)
+                                        : (details.displayName || null)}
+                                </Text>
+                            </TappableCopy>
                             {details.login ? (
                                 <View style={[styles.mb6, styles.detailsPageSectionContainer]}>
                                     <Text style={[styles.formLabel, styles.mb2]} numberOfLines={1}>
                                         {translate(isSMSLogin
                                             ? 'common.phoneNumber'
                                             : 'common.email')}
                                     </Text>
-                                    <Text style={[styles.textP]} numberOfLines={1}>
-                                        {isSMSLogin
-                                            ? toLocalPhone(getPhoneNumber(details))
-                                            : details.login}
-                                    </Text>
+                                    <TappableCopy
+                                        type={isSMSLogin ? 'phone' : 'email'}
```

Comment:
Use LOGIN_TYPE PHONE and EMAIL here https://github.com/parasharrajat/Expensify.cash/blob/66341fdeaf2b75befb00e9b6a0c935d50bd16aca/src/CONST.js#L227